### PR TITLE
Searching for class comments no longer throws error, fixes #113

### LINKED ISSUE #113: Searching for class comments throws error
When I search for a class comment with the which-Keyword Algernon throws an error.

### PR DESCRIPTION
**Pull request by BenBals**

### CODE REVIEW

**Comment by BenBals** [comment]
This pr is ready to merge from my perspective. Please review.

@robertkirsten does this fix the issue for you? (If yes then please approve, you don't need to look at the code, it's not that much.)

File: packages/AlgernonTests.package/ALGTest.class/instance/assertResultContians..st [hunk 1]
```diff
@@ -0,0 +1,3 @@
+helpers
+assertResultContians: aString
```
Comment (sinnaj-r) [inline packages/AlgernonTests.package/ALGTest.class/instance/assertResultContians..st]: Got a typo here

Comment (sinnaj-r) [reply packages/AlgernonTests.package/ALGTest.class/instance/assertResultContians..st]: We also already got the `resultsIncludeName:` and `resultsClassIncludeName:` methods. You may want to use one of them. (Or "merge" them with your method)

File: packages/Algernon.package/ALGCommentFilter.class/instance/filter.given..st [hunk 2]
```diff
@@ -3,4 +3,6 @@ filter: anItemCollection given: aStringCollection
 
 	| commentKeyWord |
 	commentKeyWord := aStringCollection second.
-	^ anItemCollection select: [ :each | each matchesComment: commentKeyWord ]
\ No newline at end of file
+	^ (anItemCollection
+		select: [ :each | each matchesComment: commentKeyWord ])
+		do: [ :each | each fuzzyRank: self class classCommentFuzzyPenalty ]
```
Comment (sinnaj-r) [inline packages/Algernon.package/ALGCommentFilter.class/instance/filter.given..st]: When a PR is this small, the comments also become more pedantic 😃: 
You can use `select:thenDo:` here

**Review by sinnaj-r — APPROVED** [review]
Looks good. I found a two (very) small things; but feel free to merge without fixing them.

**Review by robertkirsten — APPROVED** [review]
Seems good to me too, also it seems like more results are found.

**Pull request merged**